eth/discovery.go

```diff
@@ -214,6 +213,28 @@ func (d *Discovery) Serve(ctx context.Context) (err error) {
 	}
 }
 
+func (d *Discovery) forkUpdateSubs() error {
+	// update the enr entry with the new fork
+	enrEth2Entry, err := d.cfg.enrEth2Entry()
+	if err != nil {
+		return fmt.Errorf("build enr fork entry: %w", err)
+	}
+
+	d.node.Set(enrEth2Entry)
```

Comment:
no locking needed?

Reply:
I don't think so. This is just the fork_digest that we will be sharing at the discv5.